### PR TITLE
Add key describe command to vegawallet (#329)

### LINKED ISSUE #329: Introduce `key describe` sub-command
The `key describe` sub-command should describe a key with all its data.

We should use trim the `key list` sub-command to display the public key, and its name if any.

We should use this new command to improve the E2E tests.

### PR DESCRIPTION
**Pull request by wwestgarth**
closes #329 

The output when describing a key looks as follows (but with more colours):
```
Name:              mykey
Public key:        fac2d2c7454169200273e448064242570a43a97b1869cdb3e67b628e02d54182
Algorithm Name:    vega/ed25519
Algorithm Version: 1

Key pair is: tainted
Tainting a key pair marks it as unsafe to use and ensures it will not be used to sign transactions.
This mechanism is useful when the key pair has been compromised.

Metadata:
name    | mykey
keytype | ed25519
some    | thing
```

### CODE REVIEW

File: cmd/key_describe.go [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+	"code.vegaprotocol.io/vegawallet/cmd/flags"
+	"code.vegaprotocol.io/vegawallet/cmd/printer"
+	"code.vegaprotocol.io/vegawallet/wallet"
+	"code.vegaprotocol.io/vegawallet/wallets"
+	"github.com/spf13/cobra"
+)
+
+var (
+	describeKeyLong = cli.LongDesc(`
+		Describe the all known information about the specified key pair
```
Comment (jeremyletang) [inline cmd/key_describe.go:17]: English is not my first language, but I'm not sure `Describe the all know information` is what you wanted to write?

Comment (wwestgarth) [reply cmd/key_describe.go]: It certainly was not!

**Review by jeremyletang — COMMENTED** [review]
Looks pretty good, just a couple of linter issues to address + I'll let @ValentinTrinque  do a proper review there.

File: cmd/flags/passphrase.go [hunk 2]
```diff
@@ -50,8 +50,9 @@ func ReadPassphraseFile(passphraseFilePath string) (string, error) {
 		return "", fmt.Errorf("couldn't read passphrase file: %w", err)
 	}
 
-	// user might have added \n at the end of the line, let's remove it.
-	cleanupPassphrase := strings.Trim(string(rawPassphrase), "\n")
+	// user might have added a newline at the end of the line, let's remove it,
+	// remembering Windows does things differently
```
Comment (ValentinTrinque) [inline cmd/flags/passphrase.go:54]: 👍🏻

File: cmd/key_annotate.go [hunk 3]
```diff
@@ -14,14 +14,14 @@ import (
 
 var (
 	annotateKeyLong = cli.LongDesc(`
-		Add metadata to a key pair. The metadata are not updated in place, they are
-		replaced.
+		Add metadata to a key pair. All existing metadata is removed and replaced
```
Comment (ValentinTrinque) [inline cmd/key_annotate.go:17]: 👍🏻

File: tests/sc_annotate_key_test.go [hunk 4]
```diff
@@ -16,14 +16,17 @@ func TestAnnotateKey(t *testing.T) {
 
 	walletName := vgrand.RandomStr(5)
 
-	// when
-	generateKeyResp, err := KeyGenerate(t, []string{
+	cmd := []string{
```
Comment (ValentinTrinque) [inline tests/sc_annotate_key_test.go:19]: 👍🏻

**Review by ValentinTrinque — CHANGES_REQUESTED** [review]
Very good ! We are just missing the entry in the `CHANGELOG.md` and we are good to go !

**Review by ValentinTrinque — APPROVED** [review]

**Pull request merged**